cmd/root.go

```diff
@@ -11,7 +12,7 @@ import (
 	"github.com/boson-project/faas"
 )
 
-var config = "~/.faas/config" // Location of the optional system-wide config file.
+var config = configPath() // Location of the optional system-wide config file.
```

Comment:
This line can now be removed, as it's been correctly updated to be directly inlined into the `configPath` function using `filepath` to make it OS agnostic.

Indeed it _should_ probably be removed as the `configPath()` static function should be used preferentially to a global variable.